.expeditor/scripts/finish_release/publish_to_chocolatey.ps1

```diff
@@ -37,13 +37,18 @@ Write-Host "--- Publishing package"
 
 $valid_build_creator="Chef Expeditor"
 
+$pack_cmd = "choco pack $tempDir/habitat.nuspec --version $Version"
+$publish_cmd = "choco push habitat.$Version.nupkg -k $env:CHOCO_API_KEY --timeout 600"
+
 if($env:BUILDKITE_BUILD_CREATOR -eq $valid_build_creator) {
```

Comment:
I'd recomend putting this `if` block in a `try` and the below `Remove-Item` in a `finally`

Reply:
Ooooh. I am happy to learn more better patterns.